index.html

```diff
@@ -0,0 +1,261 @@
+<html>
+<head>
+  <style>
+  table {
+      border-collapse: collapse;
+      table-layout: fixed;
+  }
+  
+  table, th, tr, td{
+      border: 1px solid black;
+      height: 30px;
+  }
+  
+  table, th{
+      width: 100%;
+      font-size: 25px;
+  }
+  
+  th, a{
+     cursor: default; 
+  }
+  
+  .inputField {
+      width: 100%;
+      padding: 0px;
+      border-width: 0;
+      height: 100%;
+      font-size: 17px;
+  }
+  
+  .btn{
+      text-align: center;
+      vertical-align: middle;
+      cursor: default;
+  }
+  
+  .center {
+      margin-bottom: 8px;
+  }
+  
+  .checkbox{
+      cursor: default;
+  }
+  
+  [name=DateCell], [name=DeleteCell]  {
+      font-size: initial;
+  }
+  
+  #addRowBtn {
+      height: 25px;
+      font: icon;
+  }
+  </style>
+</head>
+
+<body onload='myTable()'>
```

Comment:
That won't work as the event name is `'load'` and not `'onload'` (just like there's `<a onclick=...` but `a.addEventListener('click', ...`